std_msgs/Header header

bool is_emg
bool is_front_safety
bool is_back_safety
bool start_button
bool stop_button